tests/org.aposin.mergeprocessor.test/src/org/aposin/mergeprocessor/configuration/svn/SVNMergeUnitTest.java

```diff
@@ -186,6 +186,90 @@ public void testGetMessageWhenLogThrowsException() {
 		assertTrue(message.trim().equals("MP [1:2] V18.0 -> V19.0"));
 	}
 
+	/**
+	 * Tests changes in Equals/hashCode methods
+	 */
+	@Test
+	public void testEquals() {
+		final SVNMergeUnit mergeUnit = new SVNMergeUnit(null, null, null, null, 1l, 2l,
+				"https://my.svn.repository.com/hugo", "https://my.svn.repository.com/franz", null, 0l, null, null, null,
+				null, null, new SvnClientMock() {
+
+					@Override
+					public List<SvnLog> log(URL url, long fromRevision, long toRevision, String author)
```

Comment:
Maybe you can create an internal static class for the SvnClientMock. Then the log method is not duplicated for each test.

Reply:
Done.